apps/etterlatte-samordning-vedtak/src/test/kotlin/no/nav/etterlatte/samordning/vedtak/SamordningVedtakRouteTest.kt

```diff
@@ -224,7 +223,6 @@ class SamordningVedtakRouteTest {
         }
 
         @Test
-        @Disabled
```

Comment:
Jeg merget nå der jeg fjernet disabled her så du kan bare ta de bort her for å unngå konflikt